clientmods/preview/init.lua

```diff
@@ -79,7 +79,7 @@ core.register_on_item_use(function(itemstack, pointed_thing)
 		return false
 	end
 
-	local pos = vector.add(core.localplayer:get_pos(), core.camera:get_offset())
+	local pos = core.camera:get_pos()
```

Comment:
These are not equal in the two 3rd perspective modes. This code is not so important, so it might suffice to note that this shootline starts from the camera, rather than the head position.

Reply:
This is just debug and testing code.